.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -1268,6 +1268,6 @@ stages:
     condition: ne(variables.SKIP_E2E_TESTS, 'true')
     variables:
       VHD_BUILD_ID: $(Build.BuildId)
-      LOCATION: $(PACKER_BUILD_LOCATION)
+      LOCATION: $(E2E_LOCATION)
```

Comment:
I don't really like we rename and override the variable multiple times before it reaches the test.
It's quite tricky to track.

Why can't we use `LOCATION` somewhere else?
Can we rename LOCATION => E2E_LOCATION inside tests?

Reply:
Do we actually need to set it?

We have a default value in code. And it can be overridden with a pipeline variable if necessary